docker-compose.yml

```diff
@@ -1,5 +1,15 @@
 version: '3.1'
 services:
+  web:
+    container_name: web
+    build:
+      context: .
+      dockerfile: web.Dockerfile
+    environment:
+      REACT_APP_BACKEND_URL: "http://localhost:8063"
```

Comment:
probably you may need to update this.

Reply:
For some reason I got `"POST /undefined/init-env" Error (404): "Not found"` which should not happen with the env set here.

Reply:
hmm, this should not happen, I will double check

Reply:
The env vars were not available during build time, I will make another PR for that.